src/SDKs/AzSdk.test.reference.props

```diff
@@ -3,7 +3,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Test.HttpRecorder" Version="[1.7.0,2.0.0)" />
     <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure.TestFramework" Version="[1.6.0,2.0.0)" />
-    <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure" Version="[3.3.5,4.0.0)" />
+    <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure" Version="[3.3.6,4.0.0)" /> 
```

Comment:
@shahabhijeet I'm making `AzSdk.test.reference.props` consistent with `AzSdk.reference.props`. Otherwise I get warnings about mismatched versions during package restore.